README.md

```diff
@@ -71,4 +71,23 @@ $ poetry run pytest
 - Install `git` using `sudo yum install git`
 - Clone the app repository in `~/opt/todoapp/`
 - Checkout branch `Exercise-4` with `git checkout Exercise-4`
-- Run the ansible playbook by using `ansible-playbook playbook -i ansible_inv`
\ No newline at end of file
+- Run the ansible playbook by using `ansible-playbook playbook -i ansible_inv`
+
+## Using Docker
+There are two container variants:
+- A development variant using Flask
+- A production variant using Gunicorn
+
+Commands needed to build each variant:\
+`docker build --target production --tag todo-app:prod .`\
+`docker build --target development --tag todo-app:dev .`
+
+Commands needed to run each variant:\
+`docker run --env-file .env -p 8000:8000 todo-app:prod`\
+`docker run --env-file .env -p 5000:5000 todo-app:dev`
+
+Alternatively, for development purposes a mount bind can be set up for the `todo_app` folder using this command:\
+`docker run --env-file .env -p 5000:5000 --mount type=bind,source=\"$(pwd)\"/todo_app,target=/app/todo_app todo-app:dev`\
```

Comment:
The escaping doesn't quite work for me on Windows, I _believe_ the below command should work reasonably well cross platform, but also note that your `target` doesn't quite match the structure inside your Dockerfile, so I wasn't able to get hot-reloading working without tweaking that:
```suggestion
`docker run --env-file .env -p 5000:5000 --mount "type=bind,source=$(pwd)/todo_app,target=/todo_app/todo_app" todo-app:dev`\
```